### PR TITLE
add setup_grafana playbook

### PR DESCRIPTION
**Pull request by HVSharma12**
Add setup_grafana playbook.

### CODE REVIEW

File: examples/ansible/setup_grafana.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+# The steps include fetching the Grafana image, setting up the Grafana container, and providing access information.
+# The steps are based on https://documentation.suse.com/alp/dolomite/html/alp-dolomite/available-alp-workloads.html#task-run-grafana-with-podman
+
+- name: Setup ALP system for Grafana
+  hosts: alphost
+  become: true
+  vars:
+    workload:
+      name: grafana
+      image: registry.opensuse.org/suse/alp/workloads/tumbleweed_containerfiles/suse/alp/workloads/grafana:latest
```
Comment (pdostal) [inline examples/ansible/setup_grafana.yml:13]: Again, I see no point of defining static content as variables.

File: examples/ansible/setup_grafana.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+# Ansible Playbook: Setup Grafana on SUSE ALP Dolomite
+# Description: This Ansible playbook automates the deployment of Grafana on a SUSE ALP Dolomite host.
+# The steps include fetching the Grafana image, setting up the Grafana container, and providing access information.
+# The steps are based on https://documentation.suse.com/alp/dolomite/html/alp-dolomite/available-alp-workloads.html#task-run-grafana-with-podman
+
+- name: Setup ALP system for Grafana
+  hosts: alphost
+  become: true
```
Comment (pdostal) [inline examples/ansible/setup_grafana.yml:9]: Why would you need to be root for that?

File: examples/ansible/setup_grafana.yml (hunk 1 above)
Comment (pdostal) [inline examples/ansible/setup_grafana.yml:13]: Again, I see no point of defining static content as variables.

File: examples/ansible/setup_grafana.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    - name: Create the Grafana container from the downloaded image
+      ansible.builtin.command: >-
+        /usr/local/bin/grafana-container-manage.sh create
+      register: grafana_container_creation
+      changed_when:
+        - ('using /etc/grafana-container.conf as configuration file' in grafana_container_creation.stdout)
+
+    - name: Start the Grafana container
+      ansible.builtin.command: >-
+        /usr/local/bin/grafana-container-manage.sh start
```
Comment (pdostal) [inline examples/ansible/setup_grafana.yml]: Aren't there `systemd` units created by the `install` runlabel?

File: examples/ansible/setup_grafana.yml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+      register: grafana_container_start
+      notify: Display Grafana access information
+      changed_when:
+        - ('Up Less than a second' in grafana_container_start.stdout)
+
+  handlers:
```
Comment (pdostal) [inline examples/ansible/setup_grafana.yml]:
```suggestion
      changed_when:
        - ('Up Less than a second' in grafana_container_start.stdout)

```

No need for handler, the debug message can be just displayed at the end of every successful run.

File: examples/ansible/setup_grafana.yml (hunk 1 above)
Comment (HVSharma12) [reply examples/ansible/setup_grafana.yml:13]: My bad, I've followed the pattern for consistency, I'll make the required changes.

File: examples/ansible/setup_grafana.yml (hunk 3 above)
Comment (HVSharma12) [reply examples/ansible/setup_grafana.yml]: Yes, there is thanks for highlighting ill use systemd.

File: examples/ansible/setup_grafana.yml (hunk 2 above)
Comment (HVSharma12) [reply examples/ansible/setup_grafana.yml:9]: Needed for runlabel install and service task

Comment (pdostal) [reply examples/ansible/setup_grafana.yml:9]: You do not need to be root to create variable in Ansible.
Variables are created locally, the `ansible.builtin.vars` plugin does not even connect to the host.

File: examples/ansible/setup_grafana.yml (hunk 1 above)
Comment (pdostal) [reply examples/ansible/setup_grafana.yml:13]: Consistency is good argument. Sorry, my bad.
Feel free to put it back.

File: examples/ansible/setup_grafana.yml (hunk 2 above)
Comment (HVSharma12) [reply examples/ansible/setup_grafana.yml:9]: Have mentioned 'become: true', at the playbook level for it to apply to all tasks.

Comment (HVSharma12) [reply examples/ansible/setup_grafana.yml:9]: Do you suggest I use become: true at the task level? For the tasks - Start and enable Grafana using systemd and Initialize the environment as they require root permission to run.

Comment (pdostal) [reply examples/ansible/setup_grafana.yml:9]: No, just keep it there.

**Review by pdostal — APPROVED** [review]

**Review by brett060102 — APPROVED** [review]
Changes look good.

**Pull request merged**